app/models/katello/content_view.rb

```diff
@@ -95,11 +95,13 @@ class ContentView < Katello::Model
                                :repository_import,
                                :repository_export_syncable])
     }
-    scope :ignore_generated, -> {
-      where.not(:generated_for => [:repository_export,
-                                   :repository_import,
-                                   :library_export_syncable,
-                                   :repository_export_syncable])
+    scope :ignore_generated, ->(include_library_generated: false) {
```

Comment:
Can you add a simple model unit test for this scope (just ensure the params are getting set correctly)